www/locations/dallas/index.html

```diff
@@ -0,0 +1,19 @@
+{% extends "site.html" %}
+{% block title %}PyLadies dfw(dallas){% endblock %}
+{% block content %}
+    <section id="archive">
+        <h1>PyLadies dfw(dallas)</h1>
+            <article>
+                <ul class="social">
+                    <aside>
+
++                        <h4><a href="{{ get_url("locations/dallas") }}">Dallas</a> |
```

Comment:
Between lines 10 - 14, you have extra `+`s which render like so: 
![2015-04-28 at 8 26 am](https://cloud.githubusercontent.com/assets/1191069/7369473/5eb1134c-ed80-11e4-9a9f-bac868835030.png)

Please remove those `+`s.